src/components/animations/appearings.tsx

```diff
@@ -26,3 +26,26 @@ export const fadeUp = keyframes({
 export const FadeUp = styled('div')({
   animation: `${fadeUp} 300ms forwards`,
 })
+
+export const slideUp = (maxHeight: number, initialMaxHeight = 0) =>
+  keyframes({
+    from: {
+      maxHeight: initialMaxHeight,
+    },
+    to: {
+      maxHeight,
+    },
+  })
+
+export const SlideUp = styled('div')(
```

Comment:
Is this really `SlideUp` maybe more something like `AnimatedHeight` is more descriptive for its purpose? :)